content/tutorials/python/hello-world.md

```diff
@@ -0,0 +1,177 @@
+---
+title: '"Hello, World! 🎉" in Python'
+toc: true
+---
+{{< lang-selector >}}
+
+In this tutorial, you will learn how to create a topic, build a producer/consumer in Typescript, and sends a "Hello, World! 🎉" message.
+
+## Prerequisites
+
+Before starting on this tutorial, you'll need to have completed the following
+
+- Install [Python](#check-python) (**3.6** or above)
+- Have the Fluvio CLI (version  `0.7.1` or greater) installed <sup>[1]</sup>
+- Have access to a Fluvio cluster.
+
+See our [getting started] guide for more details on getting set up.
+
+[getting started]: /docs/getting-started
+
+-> [1]: If you need to, you can update the Fluvio CLI by using `fluvio update`.
+
+### Create a Topic using the Fluvio CLI
+
+In Fluvio, we send all of our messages to something called a Topic, which
+is like a category for related messages. For this tutorial, we'll create
+a topic called `hello-python` using the following command:
+
+```bash
+$ fluvio topic create hello-python
+```
+
+### Check Python
+
+A Fluvio environment for Python requires Python **3.6** or above.
+
+#### Install Python
+
+Python installation varies depending on your operating system.
+
+|   Operating System     |         Instructions           |
+|------------------------|--------------------------------|
+| MacOS                  | There's an official installer from <a href="https://www.python.org/downloads/mac-osx/" target="_blank">python.org</a> to install on **macOS** but can also download python via [`brew`](https://docs.brew.sh/Homebrew-and-Python).  |
+| Linux                  | There's an official installer from <a href="https://www.python.org/downloads/source/" target="_blank">python.org</a>. It's also available in nearly every package manager. [This guide helps with ubuntu](https://docs.python-guide.org/starting/install3/linux/).
+
+To check your python version run the command `python --version`.
+
+## Writing the Application
+
+The following sections will setup your project and walk through writing the application files.
+
+### Installing Project Dependencies
+
+Run the following script to setup your project for development:
+
+```bash
+$ mkdir fluvio-python-demo && \
+cd fluvio-python-demo && \
+python -m venv venv && \
+source venv/bin/activate && \
+pip install fluvio && \
+touch producer.py consumer.py
```

Comment:
Can we do one-command-per-line? I find it easier to read

Reply:
Ah. Now I understand what you're asking for. The way it is on the [node tutorial](https://github.com/infinyon/fluvio-website/blob/master/content/tutorials/node/hello-world.md#installing-project-dependencies) is a bit annoying to copy.